script/gen_requirements_all.py

```diff
@@ -71,20 +76,55 @@ def main():
         print('\n'.join(errors))
         return
 
-    print('# Home Assistant core')
-    print('\n'.join(core_requirements()))
-    print()
-
+    output.append('# Home Assistant core')
+    output.append('\n')
+    output.append('\n'.join(core_requirements()))
+    output.append('\n')
     for pkg, requirements in reqs.items():
         for req in sorted(requirements,
                           key=lambda name: (len(name.split('.')), name)):
-            print('#', req)
+            output.append('\n# {}'.format(req))
 
         if comment_requirement(pkg):
-            print('#', pkg)
+            output.append('\n# {}\n'.format(pkg))
         else:
-            print(pkg)
-        print()
+            output.append('\n{}\n'.format(pkg))
+
+    return ''.join(output)
+
+
+def write_file(data):
+    """ Writes the modules to the requirements_all.txt. """
+    with open('requirements_all.txt', 'w+') as file:
+        file.write(data)
+
+
+def display(data):
```

Comment:
I used `diffuse` to compare the new output against the current `requirements_all.txt` to make sure that it's a match.

Well, with my changes at the moment the `RPi.GPIO` entry is missing when executed on a x86_64 machine..